custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/dc/DcDashboardPage.kt

```diff
@@ -0,0 +1,35 @@
+package jces1209.vu.page.dashboard.dc
+
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import jces1209.vu.page.dashboard.DashboardPage
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.support.ui.ExpectedConditions
+
+class DcDashboardPage(
+    private val jira: WebJira
+) : DashboardPage(
+    driver = jira.driver)
+{
+    override fun createDashboard() {
+        driver.findElement(By.className("aui-button aui-dropdown2-trigger " +
```

Comment:
What about just to go by uri `/secure/ConfigurePortalPages.jspa?view=popular` as in other tests?

Reply:
👍 for UX focus
On the other hand, going for consistency across Cloud/DC is also valuable.
It's a judgement call without an objective correct answer.
I recommend making a decision as a team and then sticking to it in other cases as well.